pepper-apis/config/redisson-jcache-housekeeping.yaml.ctmpl

```diff
@@ -21,7 +21,7 @@ singleServerConfig:
   pingConnectionInterval: 15000
 threads: 16
 nettyThreads: 32
-codec: !<org.redisson.codec.FstCodec> {}
+codec: !<org.redisson.codec.Kryo5Codec> {}
```

Comment:
Default in this version of Redisson is [Kryo5Codec](https://github.com/redisson/redisson/wiki/4.-data-serialization), with no option for FstCodec.